src/allmydata/windows/fixups.py

```diff
@@ -217,7 +217,12 @@ def writelines(self, lines):
     # Instead it "mangles" or escapes them using \x7F as an escape character, which we
     # unescape here.
     def unmangle(s):
-        return re.sub(u'\\x7F[0-9a-fA-F]*\\;', lambda m: unichr(int(m.group(0)[1:-1], 16)), s)
+        return re.sub(
+            u'\\x7F[0-9a-fA-F]*\\;',
+            # type ignored for 'unichr'
```

Comment:
maybe mention it is ignored because it doesn't exist on Python 3?  I assume this (and other Python 2/3 ignores) will go away as the porting effort reaches this code.  It might be nice to help the porter who gets to such code remember to remove the ignore somehow.

Reply:
Fixed in efd0aef28.